testdata/early_return_scope_jump.go

```diff
@@ -0,0 +1,33 @@
+package fixtures
+
+import (
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+)
+
+// MustEncode encodes a response as JSON and logs an error if it fails
+func MustEncode[T any](w http.ResponseWriter, status int, v T) {
+	if err := Encode(w, status, v); err != nil {
+		slog.Error("Error encoding response", "err", err)
+		return
+	}
+}
+
+func (c *client) renewAuthInfo() {
+	err := RenewLease(c.ctx, c, "auth", c.authCreds, func() (*hashiVault.Secret, error) {
+		authInfo, err := c.auth(c.v)
+		if err != nil {
+			return nil, fmt.Errorf("unable to renew auth info: %w", err)
+		}
```

Comment:
Are there lines like this block in the testdata of preserve scope and allow jump.

Because, with the PR only, it's uneasy to spot what is expected when these two options are not enabled.